app/site/themes/hugo-course/layouts/_default/single.html

```diff
@@ -0,0 +1,23 @@
+{{ define "main" }}
+	<main>
+		<article>
+			<h1>{{ .Title }}</h1>
+			<time>{{ .Date.Format "02.01.2006 15:04" }}</time>
+			<div>
+				{{ .Content }}
+			</div>
+			{{ with .Params.tags }}
+			<div>
+				<ul id="tags">
+					{{ range . }}
+					<li><a href="{{ "/tags/" | relLangURL }}{{ . | urlize }}">{{ . }}</a></li>
+					{{ end }}
+				</ul>
+			</div>
+			{{ end }}
+			<div>
+				{{ template "_internal/disqus.html" . }}
```

Comment:
No need for this, right ?

Reply:
Agreed, I removed this and the tags handling above from the leaf node template just now.